### DISCUSSION TITLE
Would the bot break if List of all comics was renamed?

### DISCUSSION THREAD

**Discussion by FaviFake** [theusaf/explainXKCDBot#6] [General]
Hey @theusaf,

I was thinking of renaming "List of all comics" to something along the lines of "List of recent comics", because it's a list of the most recent comics and not all of them, which could confuse newbies that just want to view the full list. I figured I should ask you:
1. Would your bot stop working properly? If so, is there a way to future-proof the bot in case the page gets moved (I'm not a programmer)?
2. Are there other important things on the site that I haven't thought of that might break if the page becomes a redirect?

**Comment by FaviFake** [comment]
Also, side note: the instructions here seem to be very outdated (i don't see any "create" link on List of all comics), do you think you could correct it? explainxkcd.com/wiki/index.php/User:DgbrtBOT

**Reply by theusaf** [reply]
I'll look into it

**Comment by theusaf** [comment]
> Hey @theusaf,
> 
> I was thinking of renaming "List of all comics" to something along the lines of "List of recent comics", because it's a list of the most recent comics and not all of them, which could confuse newbies that just want to view the full list. I figured I should ask you:
> 
>     1. Would your bot stop working properly? If so, is there a way to future-proof the bot in case the page gets moved (I'm not a programmer)?
> 
>     2. Are there other important things on the site that I haven't thought of that might break if the page becomes a redirect?

At the moment, I believe the bot would break if the list of all comics was changed. To future-proof it, it would need to check if the page was a redirect and then read/modify the relevant page instead.

Other pages that may break the bot if changed:
* Template:LATESTCOMIC
